gengokit/httptransport/httptransport.go

```diff
@@ -6,17 +6,27 @@ import (
 	"bytes"
 	"fmt"
 	"go/format"
+	"os"
 	"strconv"
 	"strings"
 	"text/template"
 	"unicode"
 
+	log "github.com/Sirupsen/logrus"
 	"github.com/TuneLab/go-truss/deftree"
 	"github.com/TuneLab/go-truss/gengokit/clientarggen"
 	gogen "github.com/golang/protobuf/protoc-gen-go/generator"
 	"github.com/pkg/errors"
 )
 
+func init() {
+	log.SetLevel(log.InfoLevel)
+	log.SetOutput(os.Stderr)
+	log.SetFormatter(&log.TextFormatter{
+		ForceColors: true,
```

Comment:
Really this should be moved to main.go for truss, we can do the checks there and make these decisions. Probably should remove it for this pull request.